virttest/qemu_storage.py

```diff
@@ -414,6 +414,8 @@ class QemuImg(storage.QemuImg):
         "source_cache_mode": "-T",
         "target_image_format": "-O",
         "convert_sparse_size": "-S",
+        "convert_rate_limit": "-r",
+        "commit_rate_limit": "-r",
```

Comment:
what about add only one item, e.g. rate_limit: "-r", for both convert_rate_limit and commit_rate_limit is the same option

Reply:
Agree, I will update them.